### DISCUSSION TITLE
Transparent bar?

### DISCUSSION THREAD

**Discussion by idelice** [neovide/neovide#2429] [General]
How do I make this bar transparent like my window?
<img width="533" alt="Screenshot 2024-03-18 at 22 48 57" src="https://github.com/neovide/neovide/assets/59118341/8c618c43-6e75-48a5-be28-591167e97a73">

**Comment by 9mm** [comment]
@Delice0 your screenshot never uploaded

**Reply by idelice** [reply]
Great... I've fixed it now
